translitua/translit.py

```diff
@@ -112,6 +117,7 @@ class UkrainianKMU(object):
     PATTERN1 = re.compile(u"(?mu)" + u'|'.join(SPECIAL_CASES.keys()))
     PATTERN2 = re.compile(u"(?mu)" + r"\b(" +
                           u'|'.join(FIRST_CHARACTERS.keys()) + u")")
+    DELETE_PATTERN = re.compile(u"(?mu)" + u'|'.join(_DELETE_CASES))
```

Comment:
Compiling regexp with flags multiline and unicode(for python 2.x) and occureneces of _DELETE_CASES.
This will be `re.compile(u"(?mu)ь|\u0027|\u2019|\u02BC")`
And later in the code this complied regexp used replace all such occurences to empty string
`DELETE_PATTERN.sub(u"", src)`